scene/gui/button.cpp

```diff
@@ -358,7 +358,7 @@ void Button::_notification(int p_what) {
 			if (!xl_text.is_empty()) {
 				text_buf->set_alignment(align_rtl_checked);
 
-				float text_buf_width = MAX(1.0f, drawable_size_remained.width); // The space's width filled by the text_buf.
+				float text_buf_width = Math::ceil(MAX(1.0f, drawable_size_remained.width)); // The space's width filled by the text_buf.
```

Comment:
Should it be `round()` to avoid errors on the other end where `62.000000047` would become `63`?
Or is it better to err on the side of caution and have one too many pixels instead of one too few?

Reply:
An extra pixel in the text buffer should not be an issue (it's setting max. width available for the string), so always ceiling is probably safer (I'm not sure how big the error can get since a lot of code is involved in control size calculations).